code/_globalvars/lists/maintenance_loot.dm

```diff
@@ -313,6 +324,14 @@ GLOBAL_LIST_INIT(rarity_loot, list(//rare: really good items
 		/obj/item/disk/nuclear/fake = 1,
 		/obj/item/skillchip/brainwashing = 1,
 		/obj/item/tattoo_kit = 1,
+		/obj/effect/spawner/random/deadspace/rig = 1,
+		/obj/effect/spawner/random/deadspace/rig/scaf = 1,
+		) = 1,
+
+	list(//data disks for lathes
+		/obj/item/disk/data/hyper/preloaded/fabricator/omni = 1,
```

Comment:
```suggestion

```
Omni should not be on the rare list due to being able to build everything, it should only be in Oddities.